src/panels/lovelace/common/evaluate-filter.ts

```diff
@@ -1,12 +1,31 @@
 import { HassEntity } from "home-assistant-js-websocket";
+import { HomeAssistant } from "../../../types";
+
+export const evaluateFilter = (
+  hass: HomeAssistant,
+  current_entity_id: string,
+  filter: any
+): boolean => {
+  const stateObj: HassEntity = hass.states[current_entity_id];
+
+  if (!stateObj) {
+    return false;
+  }
 
-export const evaluateFilter = (stateObj: HassEntity, filter: any): boolean => {
   const operator = filter.operator || "==";
   let value = filter.value ?? filter;
   let state = filter.attribute
     ? stateObj.attributes[filter.attribute]
     : stateObj.state;
 
+  if (typeof value === "string" && value.includes(".")) {
```

Comment:
```suggestion
  if (typeof value === "string" && RegExp("(sensor|number)\.").test(value)) {
```
Or maybe something like this to better check `sensor`/`input_number`/`number` entities ?

Reply:
If you want to limit I've seen code like:

`isValidEntityId(value) && ["sensor", "input_number", "number"].includes(computeDomain(value))`

There's a lot of common entity parsing functions for things like this.

Reply:
Ok, thanks for the info.

But while I saw reading, finally I don't think it's good to limit that as == ou != operators could still work with non number entity.